src/attribute.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2017 Kisio Digital and/or its affiliates.
```

Comment:
Same on all headers:
```suggestion
// Copyright (C) 2020 Kisio Digital and/or its affiliates.
```

Reply:
Done in 9b5e1ff8c4a67d96f03a90ea202680ca94943d98.